examples/express/README.md

```diff
@@ -0,0 +1,61 @@
+# Express.js
+
+Setting up Tailwind with Express is really simple, just install Tailwind:
+
+```sh
+npm install tailwindcss --save-dev
+```
+
+Create the tailwind configuration file
+
+This command will create a tailwind.js file in your root directory
+
+```sh
+./node_modules/.bin/tailwind init tailwind.js
+```
+
+Install `postcss` if you haven't already:
+
+```sh
+npm install postcss-cli
+```
+
+Create a new postcss config file:
+
+```sh
+touch postcss.config.js
+```
+
+Add this to your new ostcss config file:
+
+```js
+const tailwindcss = require('tailwindcss')
```

Comment:
Obsolete, since the same `require('tailwindcss')` declaration is used inline, in the `module.export` block.